bindings/python/pymongoarrow/errors.py

```diff
@@ -21,3 +21,14 @@ class PyMongoArrowError(Exception):
     """Base class for all PyMongoArrow exceptions."""
 
     pass
+
+
+class ArrowWriteError(PyMongoArrowError):
+    """Error raised when we encounter an exception inserting into MongoDB"""
+
+    def __init__(self, details):
+        self._details = details
+
+    @property
+    def details(self):
```

Comment:
Let's add documentation for details and how to inspect it.

Reply:
Done.